packages/now-cli/src/commands/dev/dev.ts

```diff
@@ -37,20 +37,21 @@ export default async function dev(
   });
 
   // retrieve dev command
-  const [link, frameworks] = await Promise.all([
+  let [link, frameworks] = await Promise.all([
     getLinkedProject(output, client, cwd),
     getFrameworks(client),
   ]);
 
-  if (link.status === 'error') {
-    return link.exitCode;
+  if (link.status === 'not_linked' && !process.env.__VERCEL_SKIP_DEV_CMD) {
+    link = await setupAndLink(ctx, output, cwd, false, 'link');
+
+    if (link.status === 'not_linked') {
+      return 0;
```

Comment:
Can this happen? Should we show an error message?

Reply:
Yes, this happens when the user says "N" and it prints "Aborted"

Reply:
Got it 👍 
Then maybe add a comment that mentions this, cause it seems like it's just exiting

Reply:
I'm not sure if we want to return a 0 exit code but I couldn't find a convention, maybe @TooTallNate knows?

https://stackoverflow.com/q/53304662/266535

I also added comments.

Reply:
I'm not sure of any particular convention, but at least we should be consistent with other prompts in this CLI. My gut feeling is that `0` is fine.